docs/admin/install/docker-install/compose.md

```diff
@@ -0,0 +1,95 @@
+---
+tags:
+- administration
+- installation
+- quick-start
+- docker compose
+---
+
+!!! Warning 
+    **This image is for testing and development purposes only. Use Janssen [helm charts](../charts) for production setups.**
```

Comment:
use a full link for the charts because this won't work unless the user is reading it from the README.md